src/Admin/BlockAdmin.php

```diff
@@ -137,10 +137,10 @@ protected function configureFormFields(FormMapper $form): void
 
         $form->with('general', $generalGroupOptions);
 
-        if (false === $isComposer) {
-            $form->add('name');
-        } else {
+        if (false !== $isComposer) {
```

Comment:
We might want to change this again in the future, the composer query variable arrives with "1" when is set, and it gets "false" default value when its not.

I think it is safer this way, for now atleast.

Reply:
This one could be tested with our current suite

Reply:
This is now covered.